src/main/java/com/github/krzysiekjodlowski/ox/BoardPrinter.java

```diff
@@ -0,0 +1,31 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Generates String representation of the game board.
+ * @author Krzysztof Jodlowski
+ */
+class BoardPrinter {
+    /**
+     * @param board Board object
+     * @return string representation of the Board
+     */
+    String getBoardRepresentation(final Board board) {
+        int boardSize = board.getSize();
+        int boardCapacity = board.getBoardCapacity();
+        StringBuilder stringBuilder = new StringBuilder();
+        FieldNumber currentFieldNumber;
+
```

Comment:
Wonder how do you want to print board 10kx10k on terminal?

Reply:
Fixed - now field representation adjust automatically to last field.

Reply:
I don't think this was an issues michal was talking about ;)